content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -8,30 +8,108 @@ due to theft, damages, human errors, or other factors. As such, inventory adjust
-Inventory Adjustments`.
+View inventory adjustments page
+===============================
 
-The :guilabel:`Inventory Adjustments` page shows all products that are currently in stock. Each
-line contains the following information:
+To view the :guilabel:`Inventory Adjustments` page, navigate to :menuselection:`Inventory app -->
+Operations --> Inventory Adjustments`.
 
-- :guilabel:`Product`
-- :guilabel:`Lot/Serial Number`
-- :guilabel:`On Hand Quantity`
+On the :guilabel:`Inventory Adjustments` page, all products that are currently in stock are listed,
+with each product line containing the following information:
+
+.. image:: count_products/count-products-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+- :guilabel:`Location`: is the specific location in the warheouse where a product is stored.
+- :guilabel:`Product`: is the product whose quantity is listed on the inventory adjustment line.
+- :guilabel:`Lot/Serial Number`: is the specific tracking identifier assigned to the specific
+  product listed. It can contain letters, numbers, or a combination of both.
+- :guilabel:`On Hand Quantity`: is the quantity of the product currently recorded in the database.
+- :guilabel:`UoM`: is the *unit of measure* the product is measured in. Unless otherwise specified
+  (i.e., in **pounds** or **ounces**), the default :abbr:`UoM (unit of measure)` will be **Units**.
+- :guilabel:`Counted Quantity`: is the real quantity counted during an inventory count. This field
+  will be blank by default, but can be changed depending on if it matches the :guilabel:`On Hand
+  Quantity` or not.
+- :guilabel:`Difference`: is the difference between the :guilabel:`On Hand Quantity` and
+  :guilabel:`Counted Quantity` once an inventory adjustment has been made.
+- :guilabel:`Scheduled Date`: is the date at which a count should be made. If not otherwise
+  specified, this date will default to the 31st of December of the current year.
+- :guilabel:`User`: is the person assigned to the count in the database. This can either be the
+  person physically counting the inventory, or applying the count in the database.
+- :guilabel:`Product Category`: is the category assigned internally to a specific product. Unless
+  otherwise specified (i.e., as **Consumable** or **Rental**), the default *product category* will
+  be *All*.
+- :guilabel:`Available Quantity`: is the quantity of a specific product that is currently
+  available, based on any outstanding/uncompleted sales orders, purchase orders, or manufacturing
+  orders that might change the available quantity once fulfilled.
+- :guilabel:`Accounting Date`: is the date on which the adjustments will be accounted in the Odoo
+  *Accounting* app.
+- :guilabel:`Company`: is the company whose database these inventory adjustments are being made on.
+  The company is listed in the top right corner of the database, next to the user currently logged
+  in.
+
```

Comment:
I think that it'll be listed differently for different users depending on which columns they choose to display and which settings they have activated, so I just listed them all in case they are looking for any given one